tests/integration/piv_test.py

```diff
@@ -1347,6 +1374,11 @@ def test_piv_added_stop_time_in_the_middle(mock_rabbitmq):
 
 
 def test_piv_added_stop_time_at_the_end(mock_rabbitmq):
+    """
+    schedule is added on stop 5 departure
+    stop is added after the stop 5
+    schedule is added on stop 7 (previously 5)  arrival
```

Comment:
Not sure it's exactly that (I didn't read the code for the fixture) :
```suggestion
    departure is added on stop 5
    new stop 6 is added after the stop 5
    new stop 7 is added with only arrival
```